scripts/create-aws-guide.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs')
+const inquirer = require('inquirer')
+const dedent = require('dedent')
+
+const genFrontMatter = (answers) => {
+    let frontMatter = dedent`---
+        title: ${answers.title ? answers.title : 'Untitled'}
+        linkTitle: ${answers.title ? answers.title : 'Untitled'}
+        description: Get started with ${answers.title ? answers.title : '<Untitled>'} on LocalStack
+        ---
+        ## Introduction
+
+        // Write a paragraph with 3-4 lines about the AWS service
+
+        LocalStack supports ${answers.title ? answers.title : '<Untitled>'} via the Community/Pro/Team offering, allowing 
```

Comment:
Please adjust the template here such that it complies with our [best practices](https://github.com/localstack/docs#best-practices), i.e. each sentence should be in its own line without mid-sentence line breaks.